student-manager/src/main/java/spring/studentmanager/controller/StudentController.java

```diff
@@ -30,17 +30,11 @@
 		Student foundStudent = students.stream()
 				.filter(s->s.getId()==studentId)
 				.findFirst().orElse(null);
-		if(foundStudent != null) {
-			return new ResponseEntity<>(foundStudent, HttpStatus.OK);
-		}
-		return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+		return new ResponseEntity<>(foundStudent, HttpStatus.OK);
 	}
-	@RequestMapping(method=RequestMethod.GET, path="/students/")
+	@RequestMapping(method=RequestMethod.GET, path="/students")
 	public ResponseEntity<List<Student>> findAll() {
-		if(!students.isEmpty()) {
-			return new ResponseEntity<>(students, HttpStatus.OK);
-		}
-		return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+		return new ResponseEntity<>(students, HttpStatus.OK);
 	}
```

Comment:
You did correctly removing 404 from this method. It is always better return empty list instead of 404